### PR TITLE
Resolved: Using [UnityEditor.MenuItem] causes "Private member is unused"

### LINKED ISSUE #127: Using [UnityEditor.MenuItem] causes "Private member is unused"
#### Bug description

Original bug [VSTU: Using [UnityEditor.MenuItem] causes "Private member is unused"](https://developercommunity2.visualstudio.com/t/VSTU:-Using-UnityEditorMenuItem-cause/1294126).

A method decorated with a [UnityEditor.MenuItem] is expected to be called by Unity.

We have a suppressor analyzer that removes “private member is unused” diagnostic, but the user provided code sample that reproduces the unexpected message.

### PR DESCRIPTION
**Pull request by shreyalpandit**
Fixes #127
-Using [UnityEditor.MenuItem] causes "Private member is unused"

### CODE REVIEW

File: src/Microsoft.Unity.Analyzers.Tests/UnusedMethodSuppressorTests.cs [hunk 1]
```diff
@@ -60,8 +60,11 @@ void Foo()
 
 			var suppressor = ExpectSuppressor(UnusedMethodSuppressor.Rule)
 				.WithLocation(15, 14);
-
+
```
Comment (sailro) [inline src/Microsoft.Unity.Analyzers.Tests/UnusedMethodSuppressorTests.cs:63]: I don't think we need any formatting changes (newline/tabs) in this file :)

File: src/Microsoft.Unity.Analyzers/ContextMenuSuppressor.cs [hunk 2]
```diff
@@ -98,6 +96,7 @@ private static bool IsReportable(ISymbol symbol)
 				case IFieldSymbol fieldSymbol:
 					if (fieldSymbol.GetAttributes().Any(a => a.AttributeClass.Matches(typeof(UnityEngine.ContextMenuItemAttribute))))
 						return true;
+
```
Comment (sailro) [inline src/Microsoft.Unity.Analyzers/ContextMenuSuppressor.cs:99]: unneeded

**Review by sailro — APPROVED** [review]
LGTM. Thanks. (just several unneeded changes to formatting).

**Review by jbevain — APPROVED** [review]

**Review by shreyalpandit — COMMENTED** [review]
removed formatting changes

**Comment by jbevain** [comment]
Hi @shreyalpandit,

Thank you for your contribution!

**Pull request merged**